src/App.tsx

```diff
@@ -1,18 +1,35 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
 import './App.scss';
+import { NavLink, Outlet } from 'react-router-dom';
+import classNames from 'classnames';
 
 export const App = () => {
+  const getActiveLink = ({ isActive }: { isActive: boolean }) =>
+    classNames('navbar-item', { 'has-background-grey-lighter': isActive });
+
   return (
     <div data-cy="app">
-      <Navbar />
+      <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink className={getActiveLink} to="/">
```

Comment:
The `NavLink` component's `className` prop is correctly using the `getActiveLink` function to apply conditional styling. Ensure that the `to` prop values match the routes defined in your router configuration to avoid navigation issues.